prepare-vms/scripts/trainer-cli

```diff
@@ -333,8 +331,8 @@ run_cli() {
         describe_tag $TAG
         tag_is_reachable $TAG
         echo "You may be interested in running one of the following commands:"
-        echo "trainer ips $TAG"
-        echo "trainer deploy $TAG <settings/somefile.yaml>"
+        echo "./trainer ips $TAG"
+        echo "./trainer deploy $TAG settings.yaml"
```

Comment:
I replaced `<settings/somefile.yaml>` with `settings.yaml` because it makes it easier to copy/paste the line, from a UX perspective. Do you have a reason for wanting it like it was before?